setup.cfg

```diff
@@ -1,44 +1,44 @@
-[metadata]
-name = QuCoNot
-description = QuCoNot is a Python library for all your MCT needs.
-long_description = file: README.md
-long_description_content_type = text/markdown
-author = Adam Glos
-author_email = aglos@iitis.pl
-license_files = LICENSE.txt
-url = https://github.com/QuCoNot/QuCoNot
-classifiers =
-    Development Status :: 3 - Alpha
-    Environment :: Console
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: Apache Software License
-    Natural Language :: English
-    Operating System :: POSIX
-    Operating System :: MacOS :: MacOS X
-    Operating System :: POSIX :: Linux
-    Operating System :: Microsoft :: Windows
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: 3 :: Only
-    Topic :: Scientific/Engineering :: Physics
-
-
-[options]
-python_requires = >= 3.7
-install_requires =
-    numpy >= 1.22.3
-    qiskit >= 0.39.2
-include_package_data = True
-
-[options.extras_require]
-test = pytest>=7.2;  pytest-cov>=4.0
-
-[options.packages.find]
-exclude = test
-
-[coverage:run]
-relative_files = True
+[metadata]
+name = QuCoNot
+description = QuCoNot is a Python library for all your MCT needs.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Adam Glos

```

Comment:
I don't feel OK having only my name here. Let's add all main contributors names here.

Reply:
@adamglos92 @ozlemsalehi @ankit27kh @shraddha-aangiras @HandyKurniawan 
So, should I add these names based in alphabetical order, or I put the supervisors' names last?

Reply:
this `author = Adam Glos, Özlem Salehi, Ankit Khandelwal, Handy Kurniawan, Shraddha Aangiras`
or
`author = Ankit Khandelwal, Handy Kurniawan, Shraddha Aangiras, Adam Glos, Özlem Salehi`?

Reply:
Maybe let's use alphabetical order in here

Reply:
`author = Adam Glos, Ankit Khandelwal, Handy Kurniawan, Shraddha Aangiras, Özlem Salehi`

noted :)

Reply:
@HandyKurniawan change the names in setup.py file also

Reply:
ah thank you :)